python/fledge/services/core/api/auth.py

```diff
@@ -680,6 +680,60 @@ async def enable_user(request):
         raise web.HTTPInternalServerError(reason=str(exc), body=json.dumps({"message": msg}))
     return web.json_response({'message': 'User with ID:<{}> has been {} successfully.'.format(int(user_id), _text)})
 
+@has_permission("admin")
+async def unblock_user(request):
+    """ unblock user
+        :Example:
+            curl -H "authorization: <token>" -X PUT -d '{"unblock": "true"}' http://localhost:8081/fledge/admin/{user_id}/unblock
+    """
+    if request.is_auth_optional:
+        _logger.warning(FORBIDDEN_MSG)
+        raise web.HTTPForbidden
+
+    user_id = request.match_info.get('user_id')
+    if int(user_id) == 1:
+        msg = "Restricted for Super Admin user."
+        _logger.warning(msg)
+        raise web.HTTPForbidden(reason=msg, body=json.dumps({"message": msg}))
```

Comment:
Can we block Super Admin user? I believe NO

Reply:
No we do not currently. We should make this any user with admin right in future and allowing blocking of admin users as long as there is always one unblocked admin user. This is not for the current change however.